test/UnitTests/MSTestAdapter.PlatformServices.UnitTests/Discovery/AssemblyEnumeratorWrapperTests.cs

```diff
@@ -31,9 +33,9 @@ protected override void Dispose(bool disposing)
         }
     }
 
-    public void GetTestsShouldReturnNullIfAssemblyNameIsNull() => Verify(AssemblyEnumeratorWrapper.GetTests(null, null, out _) is null);
+    public void GetTestsShouldReturnNullIfAssemblyNameIsNull() => Verify(AssemblyEnumeratorWrapper.GetTests(null, null, new TestSourceHandler(), out _) is null);
```

Comment:
Should all the `new TestSourceHandler()` in this test class change to `_mockTestSourceHandler.Object`?